tools/scripts/licenses/inter

```diff
@@ -0,0 +1,94 @@
+Copyright (c) 2016-2020 The Inter Project Authors.
```

Comment:
I think you need some metadata... check out the other files in this directory

Reply:
is `Type: sil` acceptable? not exactly sure what needs to go there for this license...

Reply:
`sil` is fine, but I think you need to add a dictionary entry to `tools/scripts/gen-attributions.py`, where the list of known licenses is defined

Reply:
updated `tools/scrips/gen-attributions.py` to include `sil` license